internal/storage/download_test.go

```diff
@@ -210,11 +321,19 @@ func host(addr string, name string, size int64, simple bool, t *testing.T) *web
 	}
 
 	// Start HTTP server in separate goroute.
-	go func() {
-		if err := w.srv.ListenAndServe(); err != nil && err != http.ErrServerClosed {
-			t.Errorf("failed to start web server: %v", err)
-		}
-	}()
+	if secure {
+		go func() {
+			if err := w.srv.ListenAndServeTLS(cert, key); err != nil && err != http.ErrServerClosed {
+				t.Errorf("The secure server is not started: %v", err)
+			}
+		}()
+	} else {
+		go func() {
+			if err := w.srv.ListenAndServe(); err != nil && err != http.ErrServerClosed {
+				t.Errorf("Failed to start web server: %v", err)
```

Comment:
Unify the error messages of the servers, the only one difference is the "secure".